src/annotator/config/index.js

```diff
@@ -106,6 +106,11 @@ const configDefinitions = {
     defaultValue: null,
     getValue: settings => settings.clientUrl,
   },
+  contentPartner: {
+    allowInBrowserExt: false,
+    defaultValue: null,
+    getValue: getHostPageSetting,
+  },
```

Comment:
Nothing new from these changes, but typing for config across the annotator and sidebar is...challenging :). Is typing in `/types/config` _only_ for sidebar-related configuration? (Yes, I do see this typed in `types/annotator`).